apps/daimo-mobile/app.json

```diff
@@ -21,12 +23,19 @@
       "supportsTablet": true,
       "bundleIdentifier": "com.daimo",
       "buildNumber": "54",
-      "associatedDomains": ["applinks:daimo.xyz"],
+      "associatedDomains": [
+        "applinks:daimo.xyz",
+        "webcredentials:daimo.xyz"
+      ],
```

Comment:
if you open + save this file, it will auto format (if not Cmd+Shift+P > format) making the diff smaller

Reply:
done